proxy/cypress/integration/login/login.spec.js

```diff
@@ -0,0 +1,33 @@
+context("Login page", () => {
+  beforeEach(() => {
+    cy.visit("/MAAS/r");
```

Comment:
You could consider using `process.env.` variables here in case it has been configured differently.

Reply:
Done. I had to add `process.env` to the config object in `plugins/index.js`, so now all process environment variables can be used in cypress tests as `Cypress.env("VAR_NAME")`.